events.markdown

```diff
@@ -2,21 +2,15 @@
 layout: page
 title: Events
 ---
-### Upcoming Events
+<div id="event-list">
 <br/>
-{%- assign sorted = (site.events | where_exp: "item", "item.date >= site.time" -%}
-{%- for event in sorted -%}
-   [**{{event.title}}**]({{ event.url | relative_url }})
-  {{event.content}}
-{%- endfor -%}
-
+{%- for event in site.events -%}
+<div class="event-link" date="{{ event.date }}">
+<a  href="{{ event.url | relative_url }}" >{{event.title}}</a>
 <br/>
-<hr/>
+{{event.summary}}
 <br/>
-### Past Events
 <br/>
-{%- assign sorted = (site.events | where_exp: "item", "item.date < site.time" -%}
-{%- for event in sorted -%}
-  [**{{event.title}}**]({{ event.url | relative_url }})
-  {{event.content}}
+</div>                                   
 {%- endfor -%}
+</div>
```

Comment:
Somewhat concerned about file_store... Github really isn't the place for large binary files.

Reply:
@ngehlenborg requested that pdf's and the link could be stored here but that is why I left it open-ended, so that we have options.

Reply:
okey-dokey!